src/Symfony/Component/Lock/Tests/Store/PdoStoreTest.php

```diff
@@ -96,10 +94,36 @@ public function testDsn(string $dsn, string $file = null)
         }
     }
 
-    public static function provideDsn()
+    public static function provideDsnWithSQLite()
     {
         $dbFile = tempnam(sys_get_temp_dir(), 'sf_sqlite_cache');
-        yield ['sqlite:'.$dbFile.'2', $dbFile.'2'];
-        yield ['sqlite::memory:'];
+        yield 'SQLite file' => ['sqlite:'.$dbFile.'2', $dbFile.'2'];
+        yield 'SQLite in memory' => ['sqlite::memory:'];
+    }
+
+    /**
+     * @requires extension pdo_pgsql
+     *
+     * @group integration
+     */
+    public function testDsnWithPostgreSQL()
+    {
+        if (!($host = getenv('POSTGRES_HOST'))) {
```

Comment:
For all similar lines:
```suggestion
        if (!$host = getenv('POSTGRES_HOST')) {
```

Reply:
Done, https://github.com/symfony/symfony/compare/b75880ec4d38d951fe48e1612ca77bf976077d2e..cb5d832e4e9a58da300ce0a737613950114bfa34